bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/common/Exceptions.kt

```diff
@@ -0,0 +1,9 @@
+package net.corda.businessnetworks.membership.common
+
+import net.corda.businessnetworks.membership.states.Membership
+import net.corda.core.flows.FlowException
+import net.corda.core.identity.Party
+
+class CounterPartyNotAMemberException(val counterParty : Party) : FlowException("Counterparty $counterParty is not a member of this business network")
+class CounterPartyMembershipNotActive(val counterParty : Party) : FlowException("Counterparty's $counterParty membership in this business network is not active")
+class NotBusinessOperatorOnThisMembership(val membership : Membership.State) : FlowException("This node is not the business network operator of this membership")
```

Comment:
I would simplify this to 
```
NotAMemberException
MembershipNotActiveException
NotBNOException
```

Reply:
I think the word counterparty is useful here. In a world where business networks is something long established and something you generally don't have to worry about the user may be confused about "who" is not a member

Reply:
I guess it can be understood from the context, no? :) Or maybe if we call parameter counterparty?

Reply:
Suppose it can be understood easily from the message coming with it. I will rename it then